abi/examples/imports/foo.ts

```diff
@@ -0,0 +1,47 @@
+import { Abi } from "../../abi-0.2";
+
+const abi: Abi = {
+  version: "0.2",
+  imports: [
+    {
+      uri: "lib2",
+      type: "wasm",
+      id: "0",
```

Comment:
What if we use IPFS hash of the abi as id?
It's guaranteed to be unique and since content addressable no need to embed the imported abi.

Reply:
The goal of the "id" is to be a (1) small in size & (2) efficient to lookup (i.e. strcmp). The URI does not satisfy these things in my opinion.

Reply:
I believe it has its own benefits as I discussed and we should compare both and see which is a better way going forward.